### ISSUE TITLE
Add field to manage comment label

### ISSUE THREAD

**Issue by cboxL** [hove-io/navitia#1278]
In order to manage cross-reference to note, we want to add a new field to the "Comment" object : _comment_label_

This text field contains the letter(s) or symbol used in the cross-reference, for instance to comment a trip in Timetable. This allows the data provider to choose the label reffering to a certain kind of note.

This issue concerns the NTFS file : _comment.txt_ and the _note_ API object.

E.g. 
{
- **id**:114
- **type**:note
- **label**:'e'
- **value**:'Except wednesday'

}

**Comment by prhod** [comment]
ok for me with the comment_label field in the comments.txt file. I'll add it soon.

**Comment by prhod** [comment]
PR https://github.com/CanalTP/navitia/pull/1250 is now including comment_label in the ntfs 0.6.

**Comment by vpassama** [comment]
Ok, now I have a technical question about comments : 

The comment is not a struct but a [map](https://github.com/CanalTP/navitia/blob/dev/source/ed/data.h#L119) containing id and comment text. If I change this and now declare a struct Comment in ed and navitia, containing id and comment_name, comment_label, I have to redeclare many things and I don't know if you want to handle it this way.

**Comment by antoine-de** [comment]
yes you can wrap this in a struct with a comment label and the comment (why do you need a name ?)

the comments are stored in kraken in a specific container:
https://github.com/CanalTP/navitia/blob/dev/source/type/comment_container.h#L71

**Comment by vpassama** [comment]
Okay great, I speak about comment_name because the NTFS documentation declares such names for the data columns : https://github.com/prhod/navitia/blob/ntfs_doc_0.6/documentation/ntfs/ntfs_0.6.md#commentstxt-optionnel.

I think it could be good to make the data columns correspond to the object attributes names.

**Comment by antoine-de** [comment]
i'm not fan of the 'name' for the value of the comment :wink:

since the api outputs a 'value' field maybe you can name the field `value` ?

**Comment by vpassama** [comment]
> i'm not fan of the 'name' for the value of the comment

Me neither :smile: 

If we don't use the column title from data, maybe we don't have to rename the attribute. Or maybe we could ask @prhod ?

Maybe comment_content or comment_value as suggested by @antoine-de would be better.

**Comment by prhod** [comment]
Ok with me for the comment_value field. I'll change it in the NTFS 0.6.
